src/MicrosoftClientAcceptanceTests/MicrosoftClientAcceptanceTests.csproj

```diff
@@ -10,16 +10,16 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.0.0" />
+    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.2.4" />
     <PackageReference Include="NServiceBus.Gateway.AcceptanceTests.Sources" Version="4.0.0" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.1.5" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.2" />
```

Comment:
I had to bump this to 5.2.2 because the latest patch for 3 had vulnerabilities, most of the builds for 4 also had vulnerabilities or were deprecated and this is a test project, so it shouldn't affect customers.